File: 01-09/Problem_6/problem6.js
let range = Array.from({length: 100}, (x,i) => i+1);
let squares = range.map(x => x*x)
let sum = range.reduce((a, b) => a+b)
let sum_squares = squares.reduce((a, b) => a+b)

console.log(sum*sum-sum_squares)

File: 01-09/Problem_6/Problem6.java

public class Problem6 {

    public static void main(String[] args) {
        int sum = 0;
        int sumSquares = 0;
        for (int i=1; i <= 100; i++) {
            sum += i;
            sumSquares += i*i;
        }
        System.out.println(sum*sum-sumSquares);
    }
}